site/docs/operations/docker.md

```diff
@@ -359,18 +359,65 @@ To see the default content of the files run the following command:
 docker run hazelcast/hazelcast-jet cat /opt/hazelcast-jet/config/hazelcast-jet.yaml
 ```
 
-## Use Built-in Modules
+## Image Variants
 
-The image comes with several officially supported connectors. You can
-find out the module names on the
-[Sources and Sinks](../api/sources-sinks.md#summary) page.
+Since the Jet version 4.4, Hazelcast Jet docker images come in two
+different variants, these are:
 
-To use one or more modules provide a list of comma separated names in
-`JET_MODULES` environment variable when running the `docker run`
+### Regular image: hazelcast-jet:<version>
+
+This image includes all of the extension modules in its classpath. It is
+prepared for everything to work out-of-the-box. If you are unsure about
+what your needs are, you probably want to use this one.
+
+### Slim image: hazelcast-jet:<version>-slim
+
+This image only includes Jet core engine and the minimal packages needed
+to run the Jet. It does not contain any extension modules. It should be
+used in situations where size is a concern.
+
+## Build Custom Image from Our Slim Image
+
+You can build your custom image that contains a fine-grained set of Jet
+extension modules by using our slim image. You can achieve this by
+adding our extension modules one by one on the Dockerfile that takes the
+Jet slim image as a base.
+
+First, create a file named `Dockerfile` in an empty directory:
+
+```Dockerfile
```

Comment:
I'm not sure Jet documentation should contain this section. This is basically a generic description for working with Dockerfiles. We should just reference the official documentation for that, because it's surely better than what we can write up. All you basically need to say is to "use slim Jet image as the basis of your custom Docker images (link to Docker documentation)".

Reply:
@frant-hartm @mtopolnik What do you think about it?